pkg/registry/rbac/rolebinding/policybased/storage.go

```diff
@@ -46,6 +47,14 @@ func (s *Storage) Create(ctx api.Context, obj runtime.Object) (runtime.Object, e
 		if s.superUser != "" && user.GetName() == s.superUser {
 			return s.StandardStorage.Create(ctx, obj)
 		}
+
+		// system:masters is special because the API server uses it for privileged loopback connections
```

Comment:
> I also don't think this ends up being very user-facing, since the primary use of this group is for infrastructure to infrastructure communication.

Even if it's not user facing we'll still want to warn admins that this is a special group, particularly for those of use who are pulling groups from an outside source.